apps/src/lab2/views/dialogs/ShareDialog.tsx

```diff
@@ -179,7 +184,17 @@ const ShareDialog: React.FunctionComponent<{
       : STUDENT_FEEDBACK_LINK;
   });
 
-  return (
+  return sharingDisabled() ? (
+    <Dialog
+      title={i18n.sharingDisabledTitle()}
+      description={i18n.sharingBlockedByTeacherAdvancedProjects()}
+      mode="light"
```

Comment:
Using "light" mode for now as header is not part of theme context. [Slack thread](https://codedotorg.slack.com/archives/C06JELCAPT9/p1747859916087629?thread_ts=1747238526.894379&cid=C06JELCAPT9)